client/assets/stylesheets/_main.scss

```diff
@@ -49,7 +49,7 @@ body {
 	background: var(--color-surface-backdrop);
 
 	&:has(.step-container-v2) {
-		background: #fdfdfd;
+		background: var(--stepper-background);
```

Comment:
@nuriapenya heads up that the stepper framework background is `#fdfdfd`, but that Calypso uses pure white. (`--color-surface` in the sites dashboard and `--color-surface-backdrop` in the main site dashboard, but both evaluate to `#fff`)

This PR makes some tweaks to ensure users don't notice a color change as they transition between stepper / checkout / and back to stepper again. But it would be simpler if everywhere used the same background color.

Should the stepper framework just use `--color-surface`?